views/js/runner/plugins/navigation/limitBackButton.js

```diff
@@ -0,0 +1,167 @@
+
+        name: pluginName,
+
+        /**
+         * Initialize the plugin (called during runner's init)
+         */
+        init: function init() {
+            var testRunner = this.getTestRunner();
+
+            return testRunner.getPluginStore(this.getName()).then(function (store) {
+                var disableState = false;
+                var processingMoveAction = false;
+
+                var toggleBackButton = function toggleBackButton() {
+                    var previous = testRunner.getPlugin("previous");
+
+                    disableState ? previous.disable() : previous.enable();
+                };
+
+                var createListener = function createListener($element, eventName) {
+                    var event = eventName
+                        ? eventName + "." + pluginName
+                        : defaultEventName + "." + pluginName;
+
+                    $element.on(event, function () {
+                        testRunner.itemRunner.trigger("responsechange." + pluginName);
+                    });
+                };
+
+                testRunner.before("move." + pluginName, function (e, direction) {
+                    processingMoveAction = true;
+                    if (direction === "previous" && disableState) {
+                        return Promise.reject();
+                    }
+                });
+
+                testRunner.after("enablenav." + pluginName, function () {
+                    toggleBackButton(disableState);
+                });
+
+                testRunner.after("disablenav." + pluginName, function () {
+                    if (disableState) {
+                        testRunner.trigger("enablenav");
+                    }
+                });
+
+                testRunner
+                    .after("renderitem", function (itemIdentifier, itemData) {
+                        var self = this;
+                        var itemRunner = testRunner.itemRunner;
+
+                        processingMoveAction = false;
+
+                        itemRunner.on("responsechange." + pluginName, function () {
+                            if (!processingMoveAction) {
+                                disableState = true;
+                                store
+                                    .setItem(getItemKey(itemIdentifier))
+                                    .then(toggleBackButton);
+                            }
+                        });
+
+                        _.forEach(customListeners, function (plugin) {
+                            var $element = self.getAreaBroker().getContainer().find(plugin.selector);
+
+                            createListener($element, plugin.eventName);
+                        });
+
+                        var itemInteractions = self.getAreaBroker().getContentArea().find('.qti-interaction');
+                        _.forEach(itemInteractions, function (element) {
+                            createListener($(element));
+                        });
```

Comment:
Here as well it is not need to manually loop over the elements since jQuery is used ;)

Reply:
Again....